src/components/DataTable.js

```diff
@@ -69,23 +123,36 @@ export default class DataTable extends React.Component {
         <table className={'table table-responsive ' + (wide ? styles.wide : styles.narrow)}>
           {(() => {
             if (wide) {
-              return (<thead>
-                <tr>
-                  {columns.map((col, i) => <th key={i}>{col}</th>)}
-                </tr>
-              </thead>);
+              return (
+                <thead>
+                  <tr>
+                    {columns.map((col, i) => {
+                      let arrow;
```

Comment:
What about moving this embedded logic into other methods (which would be the first step to extracting them to components, were we to go that far)? For example: https://gist.github.com/bscofield/0fb968a9f69b14955b508f919d5f2b86